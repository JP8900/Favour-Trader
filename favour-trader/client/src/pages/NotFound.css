.NotFound-text {
  text-align: center;
}