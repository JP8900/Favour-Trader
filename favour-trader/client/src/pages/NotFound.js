import React, { Component } from 'react';

import './NotFound.css';

class NotFound extends Component {

  render() {
    return (
      <div className='NotFound-text'>
        <h1>
          <p>Oops!</p>
        </h1>
        <div>
          <b>The page you're looking for does not exist :(</b>
        </div>
      </div>
    );
  }
}

export default NotFound;
