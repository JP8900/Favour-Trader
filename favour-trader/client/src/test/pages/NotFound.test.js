import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../pages/NotFound.js';
import AuthServiceMock from '../components/AuthServiceMock.js';

it('renders correctly', () => {
  const tree = renderer
    .create(<NotFound/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});