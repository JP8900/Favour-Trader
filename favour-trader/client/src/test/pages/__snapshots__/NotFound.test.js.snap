// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="NotFound-text"
>
  <h1>
    <p>
      Oops!
    </p>
  </h1>
  <div>
    <b>
      The page you're looking for does not exist :(
    </b>
  </div>
</div>
`;
